oci/layout/oci_delete_test.go

```diff
@@ -275,10 +276,46 @@ func TestReferenceDeleteImage_multipleImages_twoIdenticalReferences(t *testing.T
 }
 
 func loadFixture(t *testing.T, fixtureName string) string {
-	tmpDir := t.TempDir()
-	err := cp.Copy(fmt.Sprintf("fixtures/%v/", fixtureName), tmpDir)
+	destDir := t.TempDir()
+	srcDir := filepath.Join("fixtures", fixtureName)
+	err := filepath.WalkDir(srcDir, func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		relPath, err := filepath.Rel(srcDir, path)
+		if err != nil {
+			return err
+		}
+		destPath := filepath.Join(destDir, relPath)
+		switch d.Type() {
+		case fs.ModeDir:
+			if err := os.MkdirAll(destPath, 0o700); err != nil {
+				return err
+			}
+		case 0: // regular file
+			src, err := os.Open(path)
+			if err != nil {
+				return err
+			}
+			defer src.Close()
+			dest, err := os.OpenFile(destPath, os.O_WRONLY|os.O_CREATE|os.O_EXCL, 0o600)
+			if err != nil {
+				return err
+			}
+			if _, err := io.Copy(dest, src); err != nil {
+				_ = dest.Close()
```

Comment:
Why not just defer the dest.Close()?

Reply:
I didn’t want just `defer dest.Close()` because a failure in `Close` after writing could have been significant.

But that’s a good point — doing the error checking in a `defer` is not actually all that more complex, and better for peace of mind. Fixed.